### ISSUE TITLE
ModuleNotFoundError: No module named 'urllib2'

### ISSUE THREAD

**Issue by jedie** [NathanUrwin/cookie-cookie#8]
**$ cookiecutter gh:tuxredux/cookie-cookie** results in the end in:
```
Traceback (most recent call last):
  File "/tmp/tmp99moi9op.py", line 12, in <module>
    import urllib2
ModuleNotFoundError: No module named 'urllib2'
```

**Comment by tuxredux** [comment]
Hey @jedie :wave: 

Are you using Python 3? This project unfortunately only supports Python 2 at the moment. Please see issue #6. Python3 support is coming soon!

Thanks for your interest in cookie-cookie!

**Comment by jedie** [comment]
Yes, i use Python 3...

**Comment by nathanurwin** [comment]
Hey @jedie 

I added Python3 support to cookie-cookie's parent project [cookiecutter-git](https://github.com/nathanurwin/cookiecutter-git). I just have to port the changes downstream, but doing it manually takes some time. A need to write a script to automate it, or I could make a cookiecutter to wrap a cookiecutter... in a... cookiecutter... :thinking: :smile: 

I appreciate your feedback! I reopened the issue for the automatic [changelog](https://github.com/nathanurwin/cookie-cookie/blob/master/CHANGELOG.md) [generator](https://github.com/skywinder/github-changelog-generator); if you no longer wish to receive notifications feel free to unsubscribe from the issue. Thanks again!
